src/lwc/geTemplateBuilderService/geTemplateBuilderService.js-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="geTemplateBuilderFormLayout">
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="geTemplateBuilderService">
     <apiVersion>46.0</apiVersion>
```

Comment:
Should we update to version 47 if we are bumping the version in the config file as well?

Reply:
Updated https://github.com/SalesforceFoundation/Cumulus/pull/4770/commits/b53e38ffd117c047c64741863d38d97d20e6e7d4!